Cargo.toml

```diff
@@ -23,7 +23,7 @@ opt-level = 3
 codegen-units = 1
 
 [workspace.package]
-version = "0.1.3"
+version = "0.0.1"
```

Comment:
Any reason to change the version of the package to 0.0.1?

Reply:
Actually as I check, I do see why, though this might be best to get bumped to 0.0.2

Reply:
we should bump it to 0.0.2 when we want to do a release and have a matching tag. The tag and the release should be in sync. Also, for this to work, we needed it to be 0.0.1 so we can test another PR that tells you everything is up to date.

Reply:
marking as resolved because darius approved and without marking as resolved we can't merge unless we force merge


<img width="869" alt="Captura de ecrã 2023-05-04, às 23 27 53" src="https://user-images.githubusercontent.com/29093946/236342945-1cd0999a-2c8d-46f2-9d65-f8c90aef5f53.png">